### PR TITLE
Prepare release 1.3.0

### PR DESCRIPTION
**Pull request by pro-wh**
ya changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,12 @@
 
 ## Unreleased changes
 
-New features:
+Spotlight change:
 
 - ParaTime withdrawals now automatically set the fee that will soon be required on Emerald
   (0.00015 ROSE/TEST)
+- Warn users on ParaTime withdraws (and deposits) to accounts they don't own
+  ([#236](https://github.com/oasisprotocol/oasis-wallet-ext/pull/236)).
```
Comment (lukaw3d) [inline CHANGELOG.md:10]: I rebased and added this ^

**Review by lukaw3d — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (pro-wh) [reply CHANGELOG.md:10]: thanks!

Comment (pro-wh) [reply CHANGELOG.md:10]: I'm going to try moving this into a separate PR, where we're disabling the warning on deposit

**Comment by pro-wh** [comment]
moving the default fee thing out of spotlight :shrug:

**Review by tjanez — COMMENTED** [review]
Should we also spotlight "Brought ParaTime transaction hashing in sync with block explorer."? I think it will be quite important for users :slightly_smiling_face: .

Should we also add PR/issue numbers to all Change Log entries?

**Comment by pro-wh** [comment]
yeah I guess we can have multiple spotlight changes.

not sure about adding PR numbers. it's more steps for developers, and our GitHub based workflow creates merge commits which include the PR numbers, and those PRs link to issues where applicable

**Comment by pro-wh** [comment]
let's try adding the PR links this time

**Comment by pro-wh** [comment]
added PR links

**Comment by tjanez** [comment]
Yeah, we'll get PR/issue numbers automatically added in the future.

I did it manually the last time and would keep doing that until we have automation in place.

**Review by tjanez — APPROVED** [review]

**Pull request merged**